frontend/packages/network-attachment-definition-plugin/integration-tests/views/nad.actions.view.ts

```diff
@@ -0,0 +1,62 @@
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
```

Comment:
If we want to avoid importing between packages, instead of duplicating I would suggest to move this to `@console/shared/src/test- util/actions.ts`.
@suomiy  what do you think?

Reply:
+1 sounds good.

IMO it could make sense to create another package `@console-test-shared`  

thoughts? 
@vojtechszocs

Reply:
Moved to `@console/shared/src/test-utils/actions.view.ts`.

Reply:
shouldn't we replace the kubevirt-plugin actions as well, so that the code can be reused?

Reply:
Let this PR focus on NAD and do it in a separated PR for kubevirt-plugin, WDYT?

Reply:
Sure, let's remove the actions.view.ts from kubevirt-plugin and import from console-shared where required

Reply:
Removed `vm.actions.view.ts` in kubevirt-plugin as well.

Reply:
Sorry for responding late, this change looks good to me.

> IMO it could make sense to create another package, eg `@console-test-shared`

Introducing another package has its own complexity, I think it's fine to have common (integration) test code in `@console/shared` package, scoped under `src/test-utils`.